lib/msf/core/payload/linux/reverse_tcp.rb

```diff
@@ -29,6 +29,8 @@ def generate
       port:        datastore['LPORT'],
       host:        datastore['LHOST'],
       retry_count: datastore['ReverseConnectRetries'],
+      sleep_seconds:  datastore['SleepSeconds'],
```

Comment:
I wonder if we should try rename these to be consistent.
It's known as `StagerRetryWait` here: https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/payload/windows/reverse_http.rb#L33
and on Android it's `SessionRetryWait` (for both established sessions and the payload).